src/index.html

```diff
@@ -11,7 +11,188 @@
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="section">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname"
+            class="input"
+            type="text"
+            autocomplete="off"
+            name="surname"
+          >
+        </div>
+
+        <div class="section">
+          <label for="name">Name:</label>
+          <input
+            id="name"
+            class="input"
+            type="text"
+            name="name"
+            minlength="3"
+            maxlength="15"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="section">
+          <label for="age">
+            How old are You?
+          </label>
+          <input
+            id="age"
+            class="input"
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+        <div class="section">
+          <label for="birthday">
+            Full date of birth:
+          </label>
+          <input
+            id="birthday"
+            class="input"
+            type="date"
+            name="birthday"
+          >
+        </div>
+        <div class="section">
+          <label for="agreement">
+            I accept the term of the agreement
+          </label>
+          <input
+            id="agreement"
+            class="input"
+            type="checkbox"
+            name="agreement"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="section">
+          <label for="email">
+            E-mail:
+          </label>
+          <input
+            id="email"
+            class="input"
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="section">
+          <label for="password">
+            Password:
+          </label>
+          <input
+            id="password"
+            class="input"
+            type="password"
+            name="password"
+          >
+        </div>
+
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="section">
+          Do you love cats?
+          <input
+            class="input"
+            type="radio"
+            name="cats"
+            value="yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+            class="input"
+            type="radio"
+            name="cats"
+            value="no"
+          >
+          <label for="no">No</label>
```

Comment:
if the label doesn't contain input, this input should have `id `with the same value as `label`'s for to make focusing work correct

Reply:
These labels are correct, as i can see. They are just standing after the inputs.